src/components/view-toggle/view-toggle.js

```diff
@@ -70,23 +70,18 @@ class ViewToggle extends LitElement {
 			button[aria-pressed="true"]:hover, button[aria-pressed="true"]:focus {
 				box-shadow: inset 0 0 0 2px #ffffff;
 			}
+
 			:host {
-				display: flex;
-				width: 100%;
+				display: block;
+				margin: 0 -0.9rem;
```

Comment:
I know this was existing before, but I'm pretty confused as to why it would use a negative margin on the host and then a positive inverse one on the container. 😕

Reply:
Yeah, very weird. Only other time I've done something like that was to allow focus styles to overflow, which doesn't really apply here since the focus styles go inwards 🤔

Reply:
Yeah wasn't sure what was going on there 😕

Reply:
Interesting, removed the margin rules and got some vdiff changes https://github.com/BrightspaceUI/labs/pull/621/files

Reply:
That is a bit weird, but I think it's OK. If I play around in the demo and toggle between your branch and `main`, the actual width is the same.

One thing that'll need fixing though is the gap between the label and the first button has shrunk to nothing, whereas before I think it was relying on whitespace between the `<span>` and the `<button>`. I think you can play with either a `margin-inline-end` on the `span` or a custom `gap` to remove that change.